smallcloudai/refact-chat-js
# Telemetry service

# Telemetry service.



## Description



Adds the telemetry calls as a service that can be called through middleware or with a hook.

## Type of change



## How to Test

- Step 1: Install redux-dev tools
- Step 2: trigger actions that should send telemtry.
- Step 3: check the actions or network and there should be some requests.

## Screenshots (if applicable)


## Checklist




## Linked Issues


## Additional Notes
